content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -181,6 +181,7 @@ different Kubernetes components.
 | `ValidateProxyRedirects` | `true` | Beta | 1.14 | |
 | `VolumeCapacityPriority` | `false` | Alpha | 1.21 | - |
 | `WarningHeaders` | `true` | Beta | 1.19 | |
+| `WarningHeaders` | `true` | GA | 1.22 | |
```

Comment:
I'm afraid you need to manually move this to the other (graduated features) table.

Reply:
oops, done